newIDE/app/src/GameDashboard/GameAnalyticsPanel.js

```diff
@@ -209,6 +248,20 @@ export const GameAnalyticsPanel = ({
                       }{' '}
                       minutes
                     </Trans>
+                    <Tooltip
+                      title={
+                        <Text>
+                          <Trans>
+                            Average of players still active after 15 minutes.
+                            This graph shows how long players stay in the game
+                            after X minutes. It helps to see if the players quit
+                            quickly or keep playing for a while.
+                          </Trans>
+                        </Text>
+                      }
+                    >
+                      <InfoIcon />
+                    </Tooltip>
```

Comment:
I think this graph is obvious. If people don't know how to read axes, I doubt an explanation will help them.
```suggestion
```